www/main/navigator/misc/components/RecordPane.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useStoreState, useStoreActions } from 'easy-peasy';
+import { ipcRenderer } from 'electron/renderer';
+import anvaad from 'anvaad-js';
+
+import { generateSRT } from 'subtitle-generator';
+
+const path = require('path');
+const remote = require('@electron/remote');
+const { i18n } = remote.require('./app');
+const fs = require('fs');
+
+export const RecordPane = ({ className }) => {
+  const { isSubtitleRecording, subtitleData } = useStoreState((state) => state.navigator);
+  const { setIsSubtitleRecording } = useStoreActions((state) => state.navigator);
+  const [fileList, setFileList] = useState([]);
+
+  const userData = remote.app.getPath('userData');
+  const directory = path.resolve(userData, 'subtitles');
+
+  const startRecording = () => {
+    if (!isSubtitleRecording) {
+      setIsSubtitleRecording(true);
+    }
+  };
+
+  const stopRecording = () => {
+    const parsed = subtitleData.map((data, index) => {
+      return {
+        id: index,
+        seconds: data.seconds,
+        content: anvaad.unicode(data.content),
+      };
+    });
+    const srt = generateSRT(parsed, 'seconds');
+    if (isSubtitleRecording) {
+      setIsSubtitleRecording(false);
+    }
+    if (!fs.existsSync(directory)) {
+      fs.mkdirSync(directory);
+    }
+    const currentDate = new Date();
+    const filename = `${currentDate.getFullYear()}-${currentDate.getMonth()}-${currentDate.getDate()}-${currentDate.getHours()}${currentDate.getMinutes()}${currentDate.getSeconds()}.srt`;
+    const filePath = path.resolve(directory, filename);
+
+    fs.writeFile(filePath, srt, updateList);
+  };
+
+  const updateList = () => {
+    fs.readdir(directory, (error, files) => {
+      const sortedFiles = files
+        .map((fileName) => ({
+          name: fileName,
+          time: fs.statSync(`${directory}/${fileName}`).mtime.getTime(),
+        }))
+        .sort((a, b) => b.time - a.time)
+        .map((file) => file.name);
+      setFileList(sortedFiles);
+    });
+  };
+
+  useEffect(() => {
+    updateList();
+  });
+
+  const download = (fileName) => {
+    const filePath = path.resolve(directory, fileName);
+    ipcRenderer.send('download-subtitle', filePath);
+  };
+
+  const deleteFile = (filename) => {
+    const filePath = path.resolve(directory, filename);
+    fs.unlink(filePath, (err) => {
+      if (err) throw err;
+    });
+    updateList();
+  };
+
+  return (
+    <div className={`subtitle-pane ${className}`}>
+      <h3>Record Subtitles</h3>
+      <p>To generate subtitles as you go through the shabads, click on the record button below:</p>
```

Comment:
veerji static strings should come from i18n. There are a lot of other places in this file which needs this issue to be fixed.